PurchasesCoreSwift/Attribution/AdClientProxy.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  AdClientProxy.swift
+//  PurchasesCoreSwift
+//
+//  Created by Juanpe Catalán on 14/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+// TODO(Post-migration): switch this back to internal the class and all these protocols and properties.
+
+public typealias AttributionDetailsBlock = ([String: Any]?, Error?) -> Void
```

Comment:
I think this should be `([String : NSObject]?, Error?) -> Void`
(Any -> NSObject)
I'm checking the header from the iAd API:
![Screen Shot 2021-07-20 at 2 35 00 PM](https://user-images.githubusercontent.com/1304821/126398483-70eea2b5-ff40-492f-a7e2-2b0929ca25cd.png)

Reply:
makes more sense 👍🏼 

I've done the [change](https://github.com/RevenueCat/purchases-ios/pull/635/commits/c65f70fecb8092644e835774267fc7e45cf8f8f4) 🙂